lib/linters/color_variables.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const parser = require('postcss-values-parser');
+const util = require('util');
+
+module.exports = {
+    name: 'colorVariables',
+    nodeTypes: ['decl'],
+    message: '%s should be replaced with an existing variable.',
+
+    lint: function colorVariablesLinter (config, node) {
+        /**
+         * Just in case, since postcss will sometimes include the semicolon
+         * in declaration values
+         */
+        node.value = node.value.replace(/;$/, '');
+
+        const ast = parser(node.value, {
```

Comment:
Could you change this to the new [`parseValue`](https://github.com/lesshint/lesshint/blob/master/docs/developer-guide/api/utils.md#parsevalue) util (I just merged it to master).